### DISCUSSION TITLE
More than 9999 frames?

### DISCUSSION THREAD

**Discussion by Voronium** [ggarra13/mrv2#204] [Q&A] [answered]
Hi! 
Is there a limitation on how long the material loaded in MRV2 could be?
It seems I can't load longer sequence than 9999 frames!?

**Comment by BigRoy** [comment, marked as answer]
Could you share what your image sequence filenames look like?

If it's 0000-9999 and then 10000+ try and make the padding equivalent over all frames, so that it's 00000 (5 digits) from the beginning.

**Reply by Voronium** [reply]
Awesome. That was exactly the problem and after fixing the numbering now everything work smoothly!
Thank you very much!! 
8-)
